### PR TITLE
add step chain fix for reqmgr

### PR DESCRIPTION
**Pull request by ticoann**
Tested. @h4d4, Lina, could you check whether I wrote correct production/testbed nodes. 
Each thread only need to run in one vocms box in production/testbed

### CODE REVIEW

File: reqmgr2/config.py [hunk 1]
```diff
@@ -146,7 +146,18 @@
...
+    parentageFixTask = extentions.section_("parentageFixTask")
+    parentageFixTask.object = "WMCore.ReqMgr.CherryPyThreads.StepChainParentageFixTask.StepChainParentageFixTask"
+    parentageFixTask.dbs_url = data.dbs_url
+    parentageFixTask.reqmgrdb_url = "%s/%s" % (data.couch_host, data.couch_reqmgr_db)
+    parentageFixTask.parentageFixDuration = 60 * 180  # every 3 hours
+    parentageFixTask.log_file = '%s/logs/reqmgr2/parentageFixTask-%s.log' % (__file__.rsplit('/', 4)[0], time.strftime("%Y%m%d"))
+    parentageFixTask.central_logdb_url = LOG_DB_URL
+    parentageFixTask.log_reporter = LOG_REPORTER
+
+if HOST.startswith("vocms0161") or HOST.startswith("vocms0731"):
```
Comment (amaltaro) [inline reqmgr2/config.py]: add these `or HOST.startswith("vocms0117") or HOST.startswith("vocms0127"):` here too.

**Comment by amaltaro** [comment]
Lina, with this PR we also have to update this wiki page:
https://cms-http-group.web.cern.ch/cms-http-group/activity.html

since there is one new thread `ReqMgr2 parentageFixTask` and we also distributed the other threads in different backends.

File: reqmgr2/config.py (hunk 1 above)
Comment (h4d4) [reply reqmgr2/config.py]: @ticoann @amaltaro 
Yes, the nodes production/testbed are well wrote

**Comment by h4d4** [comment]
@amaltaro Yes, I'll update the twiki.

**Pull request merged**